### DISCUSSION TITLE
请问G6中的所有布局方式都可以在X6中使用吗？

### DISCUSSION THREAD

**Discussion by mtiger95** [antvis/X6#2883] [Q&A]
请问G6中的所有布局方式都可以通过@antv/layout  在X6中使用吗？

**Comment by lloydzhou** [comment]
layout可以单独使用。
通常使用的方式是，先拿到节点和边，放到layout里面布局，布局完成之后调用graph.fromJSON画图

**Reply by mtiger95** [reply]
好的.
因为在@antv/layout 库中没有找到TreeGraph的导出, 请问G6中的TreeGraph 缩进树布局如何在X6中引用？
